src/content/interop/js-interop/index.md

```diff
@@ -10,12 +10,44 @@ JavaScript apps and libraries, as well as browser APIs, using `dart:js_interop`.
 Web developers can benefit from using external JS libraries in their Dart code,
 without having to rewrite anything in Dart.
 
-{% comment %}
-## Why use JS interop?
+## Next-generation JS interop
 
-TODO: Should we have a paragraph here explaining the benefits of using interop?
-{% endcomment %}
+The Dart team [recently][] overhauled the collection of features and APIs that allow
+developers access to JavaScript and browser bindings in their Dart code.
+This next generation of web interop not only improves user experience,
+but also enables [Wasm][] support, aligning Dart with the future of the Web.
 
+The following table maps Dart's new JS and web interop solutions to
+their past counterparts:
+
+<div class="table-wrapper" markdown="1">
+
+| New static interop libraries        | Previous libraries                       |
```

Comment:
We're trying NOT to use the phrase `static interop`, right? @srujzs

Reply:
The [package:web](https://pub.dev/packages/web) pub page says "Lightweight browser API bindings built around JS static interop" in the Metadata section of the sidebar. Not saying that's the end-all, but maybe that will need to change too?

Reply:
Yeah, just alphabetical